code/modules/mob/mob_helpers.dm

```diff
@@ -627,20 +627,20 @@ proc/is_blind(A)
 			A.tumble()
 	embedded = list()
 
-/mob/proc/skill_to_evade_traps(prob_catch)
+/mob/proc/skill_to_evade_traps()
 	var/prob_evade = 0
 	if(MOVING_DELIBERATELY(src))
 		prob_evade = 25
-		prob_evade += prob_evade/stats.getMult(STAT_VIG, STAT_LEVEL_GODLIKE) - prob_evade*get_max_w_class()/ITEM_SIZE_TITANIC
+		prob_evade += prob_evade * (stats.getStat(STAT_VIG)/STAT_LEVEL_GODLIKE - get_max_w_class()/ITEM_SIZE_COLOSSAL)
 		if(stats.getPerk(PERK_SURE_STEP))
-			prob_evade += prob_evade*30/STAT_LEVEL_GODLIKE
+			prob_evade += 25*30/STAT_LEVEL_GODLIKE
 		if(stats.getPerk(PERK_RAT))
-			prob_evade *= 2
+			prob_evade *= 25
 	return prob_evade
```

Comment:
I think you could stick any hardcoded numbers to variables near var/prob_evade for readability